Site/SiteAnalyticsModule.php

```diff
@@ -503,6 +643,21 @@ protected function getFacebookPixelCommand($command)
 	}
 
 	// }}}
+	// {{{ protected function getBingUETCommand()
+
+	protected function getBingUETCommand($command)
+	{
+		if (!is_array($command)) {
+			$command = array($command);
+		}
+
+		return sprintf(
+			'window.uetq.push(%s);',
+			implode(', ', array_map('json_encode', $command))
```

Comment:
In that case the `array_map` call needs to be removed. Right now it's producing output like:

```
window.uetq.push("conversion", "purchase", 395);
```

When we should be getting output like:

```
window.uetq.push({ "ec": "conversion", "ea": "purchase",  "gv": 395});